src/Proxy/DriveItemProxy.php

```diff
@@ -255,45 +257,86 @@ public function delete()
      * Uploads a file under this folder drive item.
      *
      * @param string $name
-     *        The name.
-     * @param string|resource|\GuzzleHttp\Psr7\Stream $content
-     *        The content.
+     *        The name of file(relative file path if in same directory (or) absolute file path).
      * @param array $options
      *        The options.
      *
      * @return DriveItemProxy
      *         The drive item created.
      *
-     * @todo Support name conflict behavior.
-     * @todo Support content type in options.
      */
-    public function upload($name, $content, array $options = [])
+    public function upload($name, array $options = [])
```

Comment:
Your changes should be implemented as a separate function, perhaps `startUpload`, or `createUpload` suggesting factory methods returning upload sessions?

Reply:
done